src/CSharpTest.Net.CollectionsTest/ExampleCode.cs

```diff
@@ -17,7 +17,7 @@ private class Counts
             public int Queued = 0, Dequeued = 0;
         }
 
-        [Test]
+        //[Test]
```

Comment:
Why is the commented out?

Reply:
These tests don't seem to run locally with a clean pull of the main branch.
Reverted.